### ISSUE TITLE
Inconsistent end-point for Interop pages

### ISSUE THREAD

**Issue by gsnedders** [web-platform-tests/wpt.fyi#3121]
https://wpt.fyi/interop-2022 shows a graph ending at the end of 2022, but the scores shown above are for the latest run (currently 2023). These should probably be consistent.

This difference is more stark if you look at https://wpt.fyi/interop-2021, where the scores are notably higher than the graph shows.

Per https://twitter.com/olwinfr/status/1618721146802950144, this does appear to be causing some actual confusion among people looking at the dashboard, so we should ensure it's clear what data is being presented.

cc @web-platform-tests/interop

**Comment by foolip** [comment]
When we've talked about this in the past I've been skeptical of stopping the graphs at the end of the year, because that would make any improvements after that invisible.

But now that we have "previous focus areas" it's not totally invisible, and I think it's reasonable that the graphs _eventually_ stop updating.

To make https://wpt.fyi/interop-2021 consistent by lowering the scores now presented would be not-great I think, so I wonder if we could let that run to 2022 instead, in the graph as well?

cc @DanielRyanSmith who I've discussed this with in the past.

**Comment by DanielRyanSmith** [comment]
After making the changes to the [results-analysis](https://github.com/web-platform-tests/results-analysis) scoring, it seems that we can pretty easily delimit the any year's scores by defining the end date in which we collect runs to score (the `--to` flag in the script). Currently, [the `to` date is always today's date](https://github.com/web-platform-tests/results-analysis/blob/2fd48bfdc0c886ec36839d83e261cab7bd0fa030/interop-scoring/main.js#L400) when not explicitly defined, so runs are aggregated and added to the overall scores long after the year is over.

Basically, we can define whenever we'd like to stop accumulating the score by setting this flag for specific years that we would like to freeze.

**Comment by foolip** [comment]
@gsnedders can you file an issue in the interop repo and agenda+ it so we can decide what we want to do?
